start-sshd.sh

```diff
@@ -28,20 +30,22 @@ echo admin:*:19020:0:99999:7::: >> /etc/shadow
 # also, we cannot use $$ directly, because SSHRC is not sourced but execd
 # (so instead, we use the parent of the parent PID)
 echo 'test x$USER != xocrd && exit' >> /.ssh/rc
-echo 'parent=$(cat /proc/$PPID/stat | cut -d\  -f4)' >> /.ssh/rc
+echo 'parent=$(ps -o ppid:1= $PPID)' >> /.ssh/rc
```

Comment:
For my understanding. What are you doing with the ":1"?

Reply:
ps can yield the parent of PPID (the parent of the current process), too. But we want a specific output format: no table header, and no whitespace.